server/src/gnome/keys.rs

```diff
@@ -0,0 +1,70 @@
+use oo7::Key;
```

Comment:
this file is not really useful, move it content to gnome/crypto/mod.rs

Reply:
Done.